terraform/provider_incus.tf

```diff
@@ -0,0 +1,13 @@
+provider "incus" {
+  # Incus client library looks in ~/.config/incus for client.crt and client.key for authentication
+  generate_client_certificates = true
+  accept_remote_certificate    = true
```

Comment:
I'm going to drop these two.

`accept_remote_certificate` being `true` allows for man in the middle attacks, so we shouldn't make that the default.

`generate_client_certificates` isn't likely to be useful as if you need to generate a client certificate then you can't possibly already be trusted by the remote server, so the connection will fail.